products/pluto/pluto_build/sql/corr_zoning.sql

```diff
@@ -1,49 +0,0 @@
--- Take the zoning values from research table
```

Comment:
Just curious, why weren't these applied?

Reply:
Corrections currently don't allow replacing "null" values in terms of how the sql is written, just replacing present but incorrect values. That would be super easy to change, but per discussions with Amanda and Matt, we'd prefer to not fix district "issues" with corrections anyways as they'd sort of have to be maintained (whenever lots, zoning boundaries updated, is the correction still valid etc)